.spraygun.js

```diff
@@ -37,6 +37,7 @@ exports.setup = (projectDirectory, { chalk, shell }) => {
   shell.rm("-rf", "node_modules");
   shell.rm("-rf", ".git");
   shell.exec("git init -q");
+  shell.exec("git checkout -q -b main");
```

Comment:
By default, when you `git init` a new git repo, it initializes with no branches. A branch named `master` is created implicitly the first time you make a commit (this name is hard-coded into git). We can ensure that git uses a different default branch name by explicitly creating `main` branch before making the first commit. That's what this line does.

Reply:
Curious, do we need the -q?